samples/snippets/src/main/java/com/example/bigquerydatatransfer/ScheduleBackFill.java

```diff
@@ -19,59 +19,48 @@
 import com.google.protobuf.Timestamp;
-import com.google.protobuf.util.FieldMaskUtil;
 import java.io.IOException;
 import org.threeten.bp.Clock;
 import org.threeten.bp.Instant;
 import org.threeten.bp.temporal.ChronoUnit;
 
-// Sample to update schedule back fill for transfer config
+// Sample to run schedule back fill for transfer config
 public class ScheduleBackFill {
 
   public static void main(String[] args) throws IOException {
     // TODO(developer): Replace these variables before running the sample.
     String configId = "MY_CONFIG_ID";
     Clock clock = Clock.systemDefaultZone();
     Instant instant = clock.instant();
-    Timestamp startDate =
+    Timestamp startTime =
         Timestamp.newBuilder()
-            .setSeconds(instant.getEpochSecond())
-            .setNanos(instant.getNano())
+            .setSeconds(instant.minus(5, ChronoUnit.DAYS).getEpochSecond())
+            .setNanos(instant.minus(5, ChronoUnit.DAYS).getNano())
             .build();
-    Timestamp endDate =
+    Timestamp endTime =
         Timestamp.newBuilder()
-            .setSeconds(instant.plus(10, ChronoUnit.DAYS).getEpochSecond())
-            .setNanos(instant.plus(10, ChronoUnit.DAYS).getNano())
+            .setSeconds(instant.minus(2, ChronoUnit.DAYS).getEpochSecond())
+            .setNanos(instant.minus(2, ChronoUnit.DAYS).getNano())
             .build();
-    TransferConfig transferConfig =
-        TransferConfig.newBuilder()
-            .setName(configId)
-            .setScheduleOptions(
-                ScheduleOptions.newBuilder().setStartTime(startDate).setEndTime(endDate).build())
-            .build();
-    FieldMask updateMask = FieldMaskUtil.fromStringList(ImmutableList.of("start_time", "end_time"));
-    scheduleBackFill(transferConfig, updateMask);
+    scheduleBackFill(configId, startTime, endTime);
   }
 
-  public static void scheduleBackFill(TransferConfig transferConfig, FieldMask updateMask)
+  public static void scheduleBackFill(String configId, Timestamp startTime, Timestamp endTime)
       throws IOException {
-    try (DataTransferServiceClient dataTransferServiceClient = DataTransferServiceClient.create()) {
-      UpdateTransferConfigRequest request =
-          UpdateTransferConfigRequest.newBuilder()
-              .setTransferConfig(transferConfig)
-              .setUpdateMask(updateMask)
+    try (DataTransferServiceClient client = DataTransferServiceClient.create()) {
+      ScheduleTransferRunsRequest request =
+          ScheduleTransferRunsRequest.newBuilder()
+              .setParent(configId)
+              .setStartTime(startTime)
+              .setEndTime(endTime)
               .build();
-      TransferConfig updateConfig = dataTransferServiceClient.updateTransferConfig(request);
-      System.out.println(
-          "Schedule backfill updated successfully :" + updateConfig.getDisplayName());
+      ScheduleTransferRunsResponse response = client.scheduleTransferRuns(request);
```

Comment:
I see that https://cloud.google.com/bigquery-transfer/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs/scheduleRuns is deprecated. Does the client have a separate method for https://cloud.google.com/bigquery-transfer/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs/startManualRuns ?